packages/core/src/components/dialog/_dialog.scss

```diff
@@ -95,7 +95,7 @@ $dialog-padding: $pt-grid-size * 2 !default;
   min-height: $pt-icon-size-large + $dialog-padding;
   padding-left: $dialog-padding;
   padding-right: $dialog-padding / 4;
-  z-index: $pt-z-index-dialog-header;
```

Comment:
could leave a comment above `$pt-z-index-dialog-header` now that blueprint no longer uses it to remove in a future breaking version